src/classes/RD_RecurringDonations.cls

```diff
@@ -436,7 +433,8 @@ public class RD_RecurringDonations {
                     *********************************/
                                         
                     OppCloseDate = getNextDate(OppCloseDate, r);
-                
+
+                    // TODO: Rewrite this to use the NPSP settings for Opportunity Naming
```

Comment:
I had seen a few TODO's in the code previously, so I thought this was the practice; though in this case I noticed it early on and added the TODO to remember to discuss this particular item.

I think you're correct that our general Opp Naming does not support "N of M" in the name. What bothered me about this is that we have a define opportunity naming pattern in settings and that is used for all new Opportunities except recurring donations. If the User runs the batch job to force renaming, it will overwrite whatever name is created here with the defined naming pattern. Seems inconsistent to me. I don't think it's something we should tackle in this story, but it might be worth adding to the backlog.